repo/repo.go

```diff
@@ -29,9 +29,15 @@ func CurrentIdentity() (*Identity, error) {
 		return nil, err
 	}
 
+	signingKey, err := GetConfig("user.signingkey")
+//         if err != nil {
```

Comment:
When running 'git config user.signingkey' and no key is setup, the return code is 1.
We don't want to stop because there is no signing key but maybe there is some genuine error that I don't know about.